### PR TITLE
fixed the abrt pattern

### PR DESCRIPTION
**Pull request by tkolhar**
please review it 
thanks
 sed -i -e "s|# SSLVerify = no|SSLVerify = yes|" file.txt
 
 cat file.txt
 SSLVerify = yes

### CODE REVIEW

**Comment by Ichimonji10** [comment]
This pull request is the follow-up to #109.

**Comment by Ichimonji10** [comment]
Please rebase this pull request. There are nine pull request merges following this commit's parent.

ACK pending rebase.

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -417,7 +417,7 @@ def setup_abrt():
     # edit the config files
     run('sed -i -e "s|^URL = .*|URL = https://{0}:8443/abrt/|" '
         '/etc/libreport/plugins/ureport.conf'.format(host))
-    run('sed -i -e "s/^SSLVerify = no.*/SSLVerify = yes/" '
+    run('sed -i -e "s|# SSLVerify = no|SSLVerify = yes|" '
         '/etc/libreport/plugins/ureport.conf')
     run('sed -i -e "s/^SSLClientAuth = .*/SSLClientAuth = puppet/" '
```
Comment (elyezer) [inline automation_tools/__init__.py]: `SSLCLientAuth` will not be update, please fix its pattern too.

**Comment by elyezer** [comment]
This PR is also the follow-up to #122

**Pull request closed without merging**